Loop/Managers/LoopDataManager.swift

```diff
@@ -61,7 +63,12 @@ final class LoopDataManager {
         overrideHistory: TemporaryScheduleOverrideHistory = UserDefaults.appGroup?.overrideHistory ?? .init(),
         lastPumpEventsReconciliation: Date?,
         analyticsServicesManager: AnalyticsServicesManager,
-        localCacheDuration: TimeInterval = .days(1)
+        localCacheDuration: TimeInterval = .days(1),
+        doseStore: DoseStoreProtocol? = nil,
```

Comment:
I think the stores can be made non-optional, now that both paths (testing, and normal usage) both provide stores, as well as the backup creation logic below.

Reply:
Done! It looks so much cleaner now : )